.github/workflows/doc-build-pdf.yml

```diff
@@ -20,96 +20,109 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  # Same as in build.yml
+  TOX_ENV:                "docker-${{ github.event.inputs.platform || 'ubuntu-jammy-standard' }}-incremental"
+  BUILD_IMAGE:            "localhost:5000/${{ github.repository }}/sage-${{ github.event.inputs.platform || 'ubuntu-jammy-standard' }}-with-targets:ci"
+  FROM_DOCKER_REPOSITORY: "ghcr.io/sagemath/sage/"
```

Comment:
Does `localhost:5000` in `BUILD_IMAGE` mean that the built image is stored in the local registry?

What is the local registry for? The built image is shared among different jobs?

Reply:
It's just a very robust way to have the "docker build action" store its result, for loading it later.

It's not shared between the different jobs; there's no mechanism for doing that. Each job is separate.

Reply:
OK.